test/fixture/generatedTypeScript/interfaces/IKitchenSink.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable import/prefer-default-export, import/no-cycle, @typescript-eslint/naming-convention, @typescript-eslint/no-empty-interface */
+import { 
+  ICodeableConcept,
+  ICoding,
+  IElement,
+  IPrimitiveBoolean,
+  IPrimitiveCanonical,
+  IPrimitiveString,
+  IPrimitiveTime,
+  IPrimitiveUrl,
+} from "../internal";
+
+export interface IKitchenSink extends IElement {
```

Comment:
Codacy found an issue: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)